### ISSUE TITLE
All books show as 14KB in length

### ISSUE THREAD

**Issue by tolik2020** [renanxcortes/springerQuarantineBooksR#61]
I am having an issue downloading all the Springer books. I end up getting all the book files by running downloadspringerbook_files(); however, they are all 14KB.

**Comment by renanxcortes** [comment]
What is your `sessionInfo() `? Also, what is the Operational System you are working on? This issue was theoretically closed in https://github.com/renanxcortes/springerQuarantineBooksR/issues/53.

**Comment by renanxcortes** [comment]
This issue is duplicated with https://github.com/renanxcortes/springerQuarantineBooksR/issues/53.

**Comment by tolik2020** [comment]
> sessionInfo()
R version 4.0.0 (2020-04-24)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18363)

Matrix products: default

Random number generation:
 RNG:     Mersenne-Twister 
 Normal:  Inversion 
 Sample:  Rounding 
 
locale:
[1] LC_COLLATE=English_Canada.1252  LC_CTYPE=English_Canada.1252    LC_MONETARY=English_Canada.1252 LC_NUMERIC=C                    LC_TIME=English_Canada.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] springerQuarantineBooksR_0.1.0 tictoc_1.0                     stringr_1.4.0                  rlang_0.4.6                    readxl_1.3.1                  
[6] magrittr_1.5                   janitor_2.0.1                  httr_1.4.1                     dplyr_0.8.5                   

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.4.6      cellranger_1.1.0  compiler_4.0.0    pillar_1.4.4      prettyunits_1.1.1 remotes_2.1.1     tools_4.0.0       testthat_2.3.2   
 [9] digest_0.6.25     pkgbuild_1.0.8    pkgload_1.0.2     lubridate_1.7.8   memoise_1.1.0     tibble_3.0.1      lifecycle_0.2.0   pkgconfig_2.0.3  
[17] cli_2.0.2         rstudioapi_0.11   curl_4.3          withr_2.2.0       generics_0.0.2    desc_1.2.0        fs_1.4.1          vctrs_0.3.0      
[25] devtools_2.3.0    rprojroot_1.3-2   tidyselect_1.1.0  snakecase_0.11.0  glue_1.4.1        R6_2.4.1          processx_3.4.2    fansi_0.4.1      
[33] sessioninfo_1.1.1 callr_3.4.3       purrr_0.3.4       backports_1.1.6   ps_1.3.3          ellipsis_0.3.1    usethis_1.6.1     assertthat_0.2.1 
[41] stringi_1.4.6     crayon_1.3.4

**Comment by tolik2020** [comment]
It seems like [#53](https://github.com/renanxcortes/springerQuarantineBooksR/issues/53) is still an open issue.

**Comment by renanxcortes** [comment]
> It seems like [#53](https://github.com/renanxcortes/springerQuarantineBooksR/issues/53) is still an open issue.

Yes, I reopened it because springer reinforced its reCaptcha system.
